paragon/_overrides.scss

```diff
@@ -144,6 +181,24 @@ footer.footer {
   }
 }
 
+// Discussions mfe overwrites learning header color to white
+.learning-header {
+  background-color: $header-bg !important;
+  .course-title-lockup {
+    // Update course name color in mfe header
+    color: $main-nav-color;
```

Comment:
`main-nav-color` is optional and we don't define a default for this. So this needs to be move under the [if block here](https://github.com/open-craft/edx-simple-theme/blob/817f97d2421f92602cedff98539d9f1cb2898268/paragon/_overrides.scss#L21-L25)

Infact, do we need to define css for `.course-title-lockup` at all ? Doesn't `header span` which is defined there already cover it ?

Reply:
Yes. The `header span` change was missing from Olive and I was using it as base initially which led to this confusion. Removed this part!